spec/models/test_track/misconfiguration_notifier_spec.rb

```diff
@@ -37,31 +37,35 @@
       end
 
       context "given an Airbrake that responds to .notify_or_ignore and .notify" do
+        subject { TestTrack::MisconfigurationNotifier::Wrapper.new(TestTrack::MisconfigurationNotifier::Airbrake.new) }
+
         before do
           stub_const("Airbrake", double("Airbrake", notify: nil, notify_or_ignore: nil))
```

Comment:
There should be a test where Airbrake isn't defined for both the TestTrack.misconfiguration_notifier lazy getter and for the notifier itself, right?

Reply:
hm, i'm not sure i follow. is that covered in the test on line 34? these tests are about the MisconfiguratonNotifier class and its behavior and not the TestTrack.misconfiguration_notifier behavior. i can add tests for that separately.

Reply:
got it. line 34 wasn't visible in the diff, obviously, so I missed that that case was already covered. But you could do things like assert that Aibrake isn't defined or otherwise document that that's what's being exercised in the test title.